src/stylesheets/components/_graphic-list.scss

```diff
@@ -1,18 +1,20 @@
 // A collection of media block objects
 
 .usa-graphic_list {
+  @include typeset;
+
   .usa-graphic_list-row {
     .usa-media_block {
-      margin-bottom: $site-margins;
+      margin-bottom: units($theme-site-margins);
```

Comment:
I wonder about using `$theme-site-margins` as a bottom margin in these. Is it important that if the theme site margins change that this changes with it?

Reply:
This is similar to the section component where I asked the same thing. You recommended using that variable in a similar situation: https://github.com/uswds/uswds/pull/2727#discussion_r216104347

Reply:
I was wrong there :( You were right and I wasn't looking closely enough.
_Twice in a row!_

Reply:
Updated to unit values in 1688ff6, but we will also need to update that file.